app-backend/db/src/main/scala/SceneToLayerDao.scala

```diff
@@ -116,7 +116,7 @@ object SceneToLayerDao extends Dao[SceneToLayer] with LazyLogging {
     val select = fr"""
     SELECT
       scene_id, project_id, project_layer_id, accepted, scene_order, mosaic_definition, scene_type, ingest_location,
-      data_footprint, is_single_band, single_band_options
+      data_footprint, project_layers.is_single_band, project_layers.single_band_options
```

Comment:
This shouldn't be necessary and makes it looks like there's more than one source for `is_single_band` and `single_band_options` There's no join to projects happening anymore so the ambiguity should be resolved.